announcements/announcements.go

```diff
@@ -56,18 +56,15 @@ func Listen(ctx context.Context, im *imap.Dialer, announcements chan<- Announcem
 					continue
 				}
 
-				announcements <- Announcement{
-					Subject:  email.Subject,
-					Contents: formatContents(contents),
-				}
+				announcements <- formatAnnouncement(2000, email.Subject, contents)
```

Comment:
```suggestion
				announcements <- formatAnnouncement(1990, email.Subject, contents)
```

Reply:
2000 does not take into account extra formatting for the bolding and new lines for sending the message, which can cause problems

Reply:
max len is 2048, so it's fine already